### PR TITLE
Added Description for new Branch

### PR DESCRIPTION
**Pull request by Kaboomboom3**

### CODE REVIEW

**Comment by VladFigy** [comment]
Hello there person

**Comment by cpaca** [comment]
meow

**Comment by xX133751R4WB3RRYF4RM3R27Xx** [comment]
needs more strawberries

**Comment by NullSnapshot** [comment]
No changelog given. Please update and atomize your commits. 👍 once this is done.

File: README.md [hunk 1]
```diff
@@ -3,4 +3,6 @@ DemoRepository
 
 This is a DemoRepository
 
-d[jparhioshgfb]
\ No newline at end of file
+d[jparhioshgfb]
```
Comment (NullSnapshot) [inline README.md]: This doesn't seem like proper grammar.

**Comment by AnonymousChief04** [comment]
I AM SPECIAL

File: README.md [hunk 2]
```diff
@@ -3,4 +3,6 @@ DemoRepository
...
+
+This Commit is on new_branch
```
Comment (NullSnapshot) [inline README.md]: missing period.

File: README.md (hunk 1 above)
Comment (AnonymousChief04) [reply README.md]: SOTP

Comment (xX133751R4WB3RRYF4RM3R27Xx) [reply README.md]: Strawberry

**Comment by DavyHyun** [comment]
Yo bro its me bro

File: README.md (hunk 2 above)
Comment (AnonymousChief04) [reply README.md]: ୧༼✿ ͡◕ д ◕͡ ༽୨ 
K

**Comment by SchoolUsername** [comment]
EAT

**Comment by AnonymousChief04** [comment]
୧༼✿ ͡◕ д ◕͡ ༽୨ Hoi

File: README.md (hunk 1 above)
Comment (SchoolUsername) [reply README.md]: strawberry, strawberry, mango, mango, strawberry, strawberry

**Comment by halfbakedplans** [comment]
yeet

File: README.md (hunk 2 above)
Comment (robotslovebread) [reply README.md]: good fixing. always remember the period.

**Comment by Kaboomboom3** [comment]
Closes #2

**Pull request merged**